### ISSUE TITLE
Accessibility - no label on legal links

### ISSUE THREAD

**Issue by kyjak** [carbon-design-system/carbon-for-ibm-dotcom#6024]
The element with "complementary" role does not have a label
please note when adding labels, these need to be translated to all DOCS languages, we can help with that,
and include the label in our translation request.

`<aside class="bx--legal-nav__container" data-autoid="dds--footer-legal-nav">`

**Comment by RobertaJHahn** [comment]
@kyjak  Could you provide a link to where you are seeing this issue?  Also provide the names of the tool(s) you are using to identify this issue?  We are forwarding this bug to our accessibility team in India for review.

**Comment by kyjak** [comment]
you can use our dev https://ibmdocs-dev.mybluemix.net/docs/en
and the tool to check errors is "IBM Equal Access Accessibility Checker", plugin for chrome

**Comment by Praveen-sr** [comment]
@jeffchew @RobertaJHahn
Footer default - Landmark region ( On JAWS FF & iPhone Voiceover)
Issue: Found 2 navigation region and one complimentary region with in the footer section . Which is not needed.
Expected: Avoid resentence or give the first as primary navigational links and second one as secondary navigation.

**Comment by RobertaJHahn** [comment]
@Praveen-sr You mentioned in our meeting yesterday that this bug is a dup of another created by you.  Could you find this issue and make a note of the issue number here?  Thanks.

**Comment by Praveen-sr** [comment]
@RobertaJHahn I searched our board for the issue opened during Oct 2020, but wasn't successful. Once the issue is closed on github, we are unable to see them.

**Comment by Praveen-sr** [comment]
@RobertaJHahn Ram verified the links on Footer and they are working fine.
Here is the Screen shot of JAWS speech:
![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/ca2ffd76-d673-4ae3-ab98-83e215809dee)

**Comment by RobertaJHahn** [comment]
Closing issue based on Praveen's comments above.
